pkgs/top-level/python-packages.nix

```diff
@@ -10,15 +10,21 @@ self: super: with self; {
 
   bootstrap = lib.recurseIntoAttrs {
     flit-core = toPythonModule (callPackage ../development/python-modules/bootstrap/flit-core { });
-    installer = toPythonModule (callPackage ../development/python-modules/bootstrap/installer {
```

Comment:
Did your editor autoformatter touched this file? Or maybe nixfmt? Maybe let's just keep the earcut part 😆

Reply:
Ouch yes I ran nixfmt on it, I guess I had to do at least one big mistake :D
The formatting tests of the PR didn't succeed before, this made them succeed!
I'll revert this and leave only the earcut part indeed. Tomorrow :)

Reply:
fixed